RevenueCatUI/Templates/Example1Template.swift

```diff
@@ -133,6 +131,8 @@ private struct Example1TemplateContent: View {
         .controlSize(.large)
     }
 
+    private static let imageAspectRatio = 0.7
```

Comment:
cc @guido732 @lburdock FYI we're hardcoding the aspect ratio here for now

Reply:
Yeah once we figure out what we want to do for the "actual" template we can revisit this.